internal/future/testing/fake_frontend.go

```diff
@@ -0,0 +1,46 @@
+/*
+Copyright 2019 Google LLC
```

Comment:
What does fake_frontend.go represent here? What do you anticipate goes under future/testing folder ? E2E tests - or - unit tests for each component (i would imaging UTs living within the component folder may be in a test subfolder)

Reply:
It's used in the internal gRPC tests that are in the follow PRs. It's using the old FE def but should be updated to the new one once that's available.